packages/react-native/scripts/cocoapods/__tests__/codegen_utils-test.rb

```diff
@@ -513,9 +513,10 @@ def get_podspec_no_fabric_no_script
           'source' => { :git => '' },
           'header_mappings_dir' => './',
           'platforms' => {
-            :ios => '13.4',
+            :ios => '15.1',
```

Comment:
This was updated since the tests ran last: https://github.com/facebook/react-native/pull/46137